code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -307,6 +317,23 @@ GLOBAL_LIST_EMPTY(PDAs)
 //BASIC FUNCTIONS===================================
 
 			if("Refresh")//Refresh, goes to the end of the proc.
+			if ("Toggle_Font")
+				//CODE REVISION 2
+				font_index = (font_index + 1) % 4
+
+				switch(font_index)
+					if (0)
+						font_mode = "font-family:\"VT323\", monospace;letter-spacing:1px;"
+					if (1)
+						font_mode = "font-family:\"Share Tech Mono\", monospace;letter-spacing:0px;"
+					if (2)
+						font_mode = "font-family:\"Orbitron\", monospace;letter-spacing:0px; font-size:15px"
+					if (3)
+						font_mode = "font-family:monospace;"
+			if ("Change_Color")
+				var/new_color = input("Please enter a color name or hex value (Default is \'#808000\').")

```

Comment:
why not `as color` instead of hex?

Reply:
The input proc pulls up the color picker if you add ```as color``` after the closing parenthesis, so you don't have to type in the string or color name. Even if typing "red", or "green" into the field works, the color picker is better.

Reply:
Ohhhhh. Okay lemme do that at some point